src/libraries/System.Text.Json/tests/System.Text.Json.Tests/Serialization/Value.WriteTests.cs

```diff
@@ -112,5 +112,20 @@ public static void WritePrimitives()
                 Assert.Equal(@"""1.2.3.4""", JsonSerializer.Serialize(version));
             }
         }
+
+        [Theory]
+        [InlineData("1:59:59", "01:59:59")]
+        [InlineData("23:59:59")]
+        [InlineData("23:59:59.9", "23:59:59.9000000")]
+        [InlineData("23:59:59.9999999")]
+        [InlineData("1.23:59:59")]
+        [InlineData("9999999.23:59:59.9999999")]
+        [InlineData("-9999999.23:59:59.9999999")]
```

Comment:
Echoing https://github.com/dotnet/runtime/pull/54186/files#r651698255, might make sense to include `TimeSpan.MaxValue`/`MinValue` as inputs here.

Reply:
Updated